src/StripeTests/Entities/CreditNotes/CreditNoteTest.cs

```diff
@@ -25,14 +25,15 @@ public void Deserialize()
         [Fact]
         public void DeserializeWithExpansions()
         {
+            // TODO: Fix stripe-mock to expand customer balance transaction
             string[] expansions =
             {
               "customer",
```

Comment:
Can you add `balance_transaction` to the list?

Reply:
Can you clarify what you mean? CreditNote does not have `balance_transaction` per the docs. It does have `customer_balance_transaction` but stripe-mock does not expand this properly, hence the TODO above since I won't do it in this PR.

Reply:
Ah nevermind, I misread the diff.